zerver/lib/event_queue.py

```diff
@@ -629,6 +679,7 @@ def receiver_is_idle(user_profile_id, realm_presences):
     return off_zulip or idle
 
 def process_message_event(event_template, users):
```

Comment:
What's the difference between `str` and `text_type`?  Should I just prefer `text_type` for everything?